.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
```

Comment:
This should not have been removed

Reply:
Agreeded, we should also upgrade the orb to use version 2.x.x

Reply:
What version should we upgrade to?